.github/workflows/release_updates.yml

```diff
@@ -0,0 +1,19 @@
+# Automatically updates "vX" branches based on GitHub releases. To cut a new
+# release, use the GitHub UI where you enter a tag name and release name of
+# "vX.Y.Z". See https://github.com/jupyterhub/action-k3s-helm/releases.
+name: Release updates
+
+on:
+  release:
+    types: [published, edited]
+
+jobs:
+  actions-tagger:
+    runs-on: windows-latest
```

Comment:
I'm curious why Windows?

Reply:
Haha yeah me too, the action requires it and we have kept using it without issues.